www/src/components/shell/onboarding/ShellStatus.js

```diff
@@ -27,94 +27,106 @@ export function ShellStatus({
   }, error,
 }) {
   return (
-    <OnboardingCard>
-      <Flex
-        align="center"
-        justify="space-between"
-      >
-        <Div>
-          <P
-            body1
-            bold
-          >
-            Creating cloud shell environment
+    <Flex
```

Comment:
Is this a card container we'll reuse potentially?

Reply:
I am not sure. It is a super simple fix to center the cloud shell init card. It is used by the `OnboardingWrapper` component also, but I wasn't sure if we want to display stepper and other non-essential information here. I guess there is a high chance the design team will want to tackle this one.